.changeset/eleven-nails-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+"rdf-validate-shacl": patch
+---
+
+Use [clownface-shacl-path](https://npm.im/clownface-shacl-path) for path traversals
```

Comment:
This can have an unknown impact on performance which needs to be checked. 

@ludovicm67, do you have an idea for a GH workflow job which would measure the time of a large dataset being validated so that we can compare if a PR does not decrease performance? I found https://github.com/benchmark-action/github-action-benchmark but it's not exactly what I have in mind

Reply:
@tpluscode no, I don't have any idea yet for this.
If I found something relevant, I will share it with you :)

Reply:
Actually, later yesterday I found a fork which should be more useful for PRs: https://github.com/openpgpjs/github-action-pull-request-benchmark

They simply check out master and PR branches and compare results. WDYT?

Reply:
I don't have any strong opinion on the tool to use for this topic, I would suggest that you go with it, and it in the future we find anything better, we can always do a switch.

Regarding the fork you mention, it seems to be a bit more dedicated for PR and compare with the default branch.